packages/@vivliostyle/theme-base/css/common/meta-properties.css

```
@@ -0,0 +1,73 @@
+:root {
+  /**
+   * Typography
+   */
+  --vs-font-family: sans-serif;
+  --vs-font-feature-settings: normal;
+  --vs-font-kerning: auto;
+  --vs-font-size: 16px;
+  --vs-font-size-on-screen: var(--vs-font-size);
+  --vs-font-size-on-print: var(--vs-font-size);
+  --vs-font-stretch: normal;
+  --vs-font-style: normal;
+  --vs-font-synthesis: weight style small-caps;
+  --vs-font-variant: normal;
+  --vs-font-variation-settings: normal;
+  --vs-font-weight: 400;
+  --vs-font-weight-bold: 700;
+  --vs-hanging-punctuation: none;
+  --vs-hyphens: auto;
+  --vs-line-break: strict;
+  --vs-line-height: 1.5;
+  --vs-letter-spacing: 0;
+  --vs-overflow-wrap: anywhere;
+  --vs-tab-size: 4;
+  --vs-text-align: start;
+  --vs-text-align-last: auto;
+  --vs-text-spacing: space-first trim-end trim-adjacent ideograph-alpha
+    ideograph-numeric;
```

```suggestion
  --vs-text-spacing: auto;
```

text-spacing は基本的に auto を使うべきだと思います。

- auto = trim-start trim-end trim-adjacent ideograph-alpha ideograph-numeric
- normal = space-first trim-end trim-adjacent ideograph-alpha ideograph-numeric

space-first という値は、本当はデフォルトで行頭約物をすべて詰めるようにしたいけれど、全角括弧類が全角幅であることを前提として作られている既存のWebやEPUBコンテンツとの互換性を維持する必要があるために、デフォルでは折り返し行頭約物だけを詰めるようにする必要があるということでできたものです。

また、text-spacing の値の仕様は変更予定で trim-end trim-adjacent のキーワードはなくなります。

- https://github.com/vivliostyle/vivliostyle.js/issues/1118